Technical-Steering-Committee/best-practices/message-management.md

```diff
@@ -12,14 +12,24 @@ E - Zowe Explorer
 S - Zowe System Services
 
 ### Message Formats
-ZWEcnnnnt where:
+
+The message is construed from 9 characters. There are two similar structures that are expected within the messages. The shared parts are explained first. Then the specifics of the formats under their headings. 
 
 **ZWE** - static for Zowe messages.
 
-**c** - one of the registered Components ... (see above).  Components need the ZLC to approve the use of the component to ensure consistency across all of the projects.
+**t** - is a Type.  One of (I, E, A, W)  General conventions are **I**nform, **E**ventual Action (often construed as error), **A**ction, **W**arning or **S**evere error

```

Comment:
Few messages reporting critical system failures and critical action required maybe?